aplib.net-demo/Assets/Scripts/Player/Movement.cs

```diff
@@ -0,0 +1,147 @@
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    [SerializeField] float maxSpeed = 7;
+    [SerializeField] float acceleration = 100;
+    Vector3 horizontalVelocity;
+    Vector2 horizontalInput;
+
+    [SerializeField] float normalGravityScale = 1.7f;
+    [SerializeField] float fallingGravityScale = 2.5f;
+    float actualGravityScale;
+    float gravity;
+
+    [SerializeField] float jumpHeight = 7;
+    [SerializeField] float airMovementScale = 0.5f;
+    bool jumpPressed;
+
+    [SerializeField] LayerMask groundMask;
+    [SerializeField] float groundDrag;
+    [SerializeField] float slopeAngle = 40;
+    float playerHeight;
+    bool isGrounded;
+
+    Rigidbody rb;
+    CapsuleCollider controller;
```

Comment:
Add `private` keywords and lower dash prefix to private variables, following C# conventions.